webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx

```diff
@@ -117,9 +118,10 @@ const ModelVersionHeader: React.FC<Props> = (
       },
       {
         danger: true,
-        disabled: !isDeletable,
+        disabled: false,
```

Comment:
nit: this line can be removed

Reply:
to be clear, I'm referring to the `disabled` line

Reply:
agreed! This has been updated